src/AbpCompanyName.AbpProjectName.Application/AbpCompanyName.AbpProjectName.Application.csproj

```diff
@@ -103,6 +99,9 @@
       <HintPath>..\packages\System.Linq.Dynamic.1.0.7\lib\net40\System.Linq.Dynamic.dll</HintPath>

```

Comment:
System.Linq.Dynamic package should be removed since System.Linq.Dynamic.Core already replaces it